nessie/lib/queries.py

```diff
@@ -306,24 +306,30 @@ def get_fetched_non_advisees():
 
 
 def get_unfetched_non_advisees():
-    sql = f"""SELECT DISTINCT(en.sis_id) AS sid
-              FROM {sis_schema()}.enrollments en
-              LEFT JOIN {asc_schema()}.students ascs ON ascs.sid = en.sis_id
-              LEFT JOIN {coe_schema()}.students coe ON coe.sid = en.sis_id
-              LEFT JOIN {undergrads_schema()}.students ug ON ug.sid = en.sis_id
-              LEFT JOIN {student_schema()}.sis_api_profiles_hist_enr hist ON hist.sid = en.sis_id
+    sql = f"""SELECT DISTINCT(attrs.sis_id) AS sid
+              FROM (
+                SELECT sis_id FROM {sis_schema()}.enrollments
+                UNION SELECT sid AS sis_id FROM {sis_schema()}.basic_attributes
+              ) attrs
+              JOIN {asc_schema()}.students ascs ON ascs.sid = attrs.sis_id
```

Comment:
@paulkerschen Was the switch from a `LEFT JOIN` to a `JOIN` intentional?

Reply:
No! Good eye, thanks.

Reply:
Amended.